chat-plugins/info.js

```diff
@@ -1499,6 +1499,20 @@ var commands = exports.commands = {
 		this.sendReplyBox("<a href=\"http://www.smogon.com/forums/forums/pok%C3%A9mon-showdown.209\">Pok&eacute;mon Showdown Forums</a>");
 	},
 
+	suggestions: function (target, room, user) {
+		if (!this.canBroadcast()) return;
+		this.sendReplyBox("<a href=\"https://www.smogon.com/forums/threads/3534365/\">Make a suggestion for Pok&eacute;mon Showdown</a>");
+	},
+
+	bugreport: 'bugs',
+	bugs: function (target, room, user) {
+		if (!this.canBroadcast()) return;
+		this.sendReplyBox(
+			"Have a replay showcasing a bug on Pok&eacute;mon Showdown?<br />" +
+			"- <a href=\"https://www.smogon.com/forums/threads/3469932/\">Bug Reports</a>"
+		);
```

Comment:
If the command is used in a battle room, it would be really awesome if a ``<button name="saveReplay">Save Replay</button>`` were displayed